src/App.jsx

```diff
@@ -24,22 +25,23 @@ export const alex = {
 };
 
 export const App = () => (
-  <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">Natasha is my wife</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Olya</h2>
-      <p className="Person__partner">Maksym is my husband</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
-    </section>
-  </div>
+  <>
+    <Person
+      person={{
+        name: 'My name is Misha',
+        age: 'I am 37',
+        partner: 'Natasha is my wife',
+      }}
+    />
+    <Person
+      person={{ name: 'My name is Olya', partner: 'Maksym is my husband' }}
+    />
+    <Person
+      person={{
+        name: 'My name is Alex',
+        age: 'I am 25',
+        partner: 'I am not married',
+      }}
+    />
```

Comment:
The `Person` component should receive the entire person object as a prop, rather than individual fields. Update the code to pass the full person objects (e.g., `misha`, `olya`, `alex`) to the `Person` component. This aligns with the task requirement: "pass the whole person as a prop `<Person person={misha} />` (not individual fields)".